README.md

```diff
@@ -1116,10 +1122,60 @@ Limited cross table searches are supported.  If you desire a join across tables
           "targetTable": "table1"
         }
       ]
+    },
+    {
+      "tableName": "table2",
```

Comment:
This is the second "table2" section, there cannot be a duplicate.

Reply:
this was meant to say table3, updated